src/backend/src/controllers/organizations.controller.ts

```diff
@@ -60,4 +60,18 @@ export default class OrganizationsController {
       next(error);
     }
   }
+
+  static async updateApplicationLink(req: Request, res: Response, next: NextFunction) {
+    try {
+      const { applicationLink } = req.body;
+      const updatedOrganization = await OrganizationsService.updateApplicationLink(
```

Comment:
If you take a look at the parameters in your updateApplicationLink service method, you will see that link is the second parameter and organizationId is the third parameter. so you should switch those up on lines 69 and 70. Also instead of passing in the organizationId, just pass in the enture organization